pkg/yurtmanager/webhook/poolservice/v1alpha1/poolservice_default.go

```diff
@@ -0,0 +1,46 @@
+/*
+Copyright 2024 The OpenYurt Authors.
+Licensed under the Apache License, Version 2.0 (the License);
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an AS IS BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1alpha1
+
+import (
+	"context"
+	"fmt"
+
+	apierrors "k8s.io/apimachinery/pkg/api/errors"
+	"k8s.io/apimachinery/pkg/runtime"
+
+	"github.com/openyurtio/openyurt/pkg/apis/network/v1alpha1"
+	viplb "github.com/openyurtio/openyurt/pkg/yurtmanager/controller/loadbalancerset/viploadbalancer"
+)
+
+// Default satisfies the defaulting webhook interface.
+func (webhook *PoolServiceHandler) Default(ctx context.Context, obj runtime.Object) error {
```

Comment:
I think that may be we need to add a Service Webhook, and add service topology annotation on Service instead of PoolService.